sdk/communication/communication-messages-rest/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Release History
 
-## 1.0.0 (2024-03-07)
+## 1.0.1 (2024-03-07)
+
+Using MessagesServiceClient:
+
+- Send WhatsApp messages.
+- Get Template List.
+- Download media file from WhatsApp server for incoming media message.
+
+### Other Changes
+
+- Fixed CI pipeline to publish MS doc.
+
```

Comment:
This is probably not worth mentioning in the Changelog as it doesn't really have any impact in the library for the end users